### ISSUE TITLE
Is there a way to point Anonymizer at a directory on your hard drive?

### ISSUE THREAD

**Issue by kirbyju** [RSNA/anonymizer#2]
Or do you have to push/pull the data via DICOM transfer via a PACS or something like https://mircwiki.rsna.org/index.php?title=FileSender?

**Comment by mdevans** [comment]
Yes, please refer to the Operation section of the documentation here: https://rsna.github.io/anonymizer/

**Comment by kirbyju** [comment]
Thanks, I didn't think to check the "File" menu for importing data because I saw options related to that in other places in the UI.  

1. I suggest that you add buttons for importing from file/directory next to the Query button on the screen that shows stats/progress about the data that's been processed so that all of the import options are displayed together.  
2. I suggest adding a button for an import directory next to where you're collecting information about DICOM servers in the project config screen. 
3. It might also be worth adding the DICOM transfer options to the File menu rather than only showing the local disk import options.  This way people who click that menu will see all available import options rather than a subset.

**Comment by mdevans** [comment]
The software design ensures there is only one location to initiate an action.
The GUI standard for project and file management have been followed, once a project has been configured and opened then files can be imported via the File menu.
The QUERY and EXPORT buttons relate to DICOM **network** transfers whilst the File/Import options relate to DICOM **files** on the local file system.
